system-variables.md

```diff
@@ -909,6 +909,18 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Range: `[0, 9223372036854775807]`
 - This variable is used to set the number of retries when the DDL operation fails. When the number of retries exceeds the parameter value, the wrong DDL operation is canceled.
 
+### `tidb_ddl_flashback_concurrency` <span class="version-mark">New in v6.3.0</span>
+
+> **Warning:**
+>
+> The feature controlled by this variable is not fully functional in the current version. It is **not recommended** that you change the default value.
```

Comment:
```suggestion
> The feature controlled by this variable is not fully functional in the current TiDB version. Do not change the default value.
```

Reply:
"not recommended" or "do not",  please check it with R&D.

Reply:
`Do not` is better.